components/UnstructuredTab/UnstructuredTab.tsx

```diff
@@ -0,0 +1,85 @@
+import { ReactNode } from 'react';
+import styled from 'styled-components';
+import { useTranslation } from 'react-i18next';
+import { Svg } from 'react-optimized-image';
+import { StyledTabButton } from 'components/Tab/Tab';
+import { Col } from 'sections/gov/components/common';
+import InfoIcon from 'assets/svg/app/info-pink.svg';
+
+export type TabInfo = {
+	title: string;
+	description: string;
+	tabChildren: ReactNode;
+	key: string;
+	blue: boolean;
+	disabled?: boolean;
+};
+
+export type UnstructuredTabProps = {
+	tabData: TabInfo;
+};
+
+const StyledTabButtonExtended = styled(StyledTabButton)`
+	flex-direction: row;
+	justify-content: space-between;
+	padding: 0px 20px;
+`;
+
+const StyledTextWrapper = styled.div`
+	background: #2a0b21;
+	color: #df46fd;
+	font-size: 12px;
+	border-left: 3px solid #df46fd;
+	display: flex;
+	align-items: center;
+	padding-right: 8px;
+`;
+
+const StyledText = styled.h2`
+	color: #df46fd;
```

Comment:
I know it's a bit bad that we only have a screenshot as design for this, but I think many of these colors should exist in `/theme/colors.ts`. 
Can recommend the VSCODE "Color Highlight" extension, then it should be fairly easy to see the colors.
This one should probably be `pink`